packages/openapi-generator/src/wrapper-files/create-file.ts

```diff
@@ -41,3 +41,28 @@ ${content}
 ` + '\n'
   );
 }
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Copy a file from a given path.
+ * @param src Path to the source file.
+ * @param dest Path to the destination file
+ * @param overwrite Whether or not existing files should be overwritten.
+ */
+export async function copyFile(
```

Comment:
Maybe the file-name needs to be changed? `create-file` does not imply copying. Maybe 'fs'?

Reply:
renamed